configure.ac

```diff
@@ -1957,7 +1957,7 @@ LIBS_TEMP="$LIBS"
 unset LIBS
 LIBS="$LIBS_TEMP"
 
-ac_configure_args="${ac_configure_args} --disable-shared --with-pic --enable-benchmark=no --enable-module-recovery --enable-module-schnorrsig"
+ac_configure_args="${ac_configure_args} --disable-shared --with-pic --enable-benchmark=no --enable-module-recovery --enable-module-schnorrsig --enable-module-ecdh -enable-experimental"
```

Comment:
You can drop `-enable-experimental`. It should no-longer be required for `--enable-module-ecdh`, and note the missing `-` in any case.

Reply:
Thanks. Done.